lib/Qubit.class.php

```diff
@@ -452,4 +452,13 @@ public static function generateIdentifierFromCounterAndMask($counter, $mask)
       }
     }, $mask);
   }
+
+  public static function actionExistsForUrl($url)
```

Comment:
This method looks like a duplicate of the `QubitObject::actionExistsForUrl()` method, and it doesn't look like it's called anywhere.

Reply:
Oops... accidentally committed that.